internal/api/v1/service/delete.go

```diff
@@ -146,6 +153,8 @@ func Delete(c *gin.Context) apierror.APIErrors {
 
 			// ... And run the unbind per app and service.
 			for serviceName, serviceConfigurations := range infoMap {
+				logger.Info("unbinding of", "service", appName, serviceName)
```

Comment:
Same because of the structured logs:

```suggestion
				logger.Info("unbinding of service", "app", appName, "service", serviceName)
```

Reply:
done with commit f59a6fe6a7e2948f7004856dd35ce18afc625817